modules/apps/frontend-compatibility/frontend-compatibility-ie/src/main/java/com/liferay/frontend/compatibility/ie/internal/servlet/taglib/IETopHeadDynamicInclude.java

```diff
@@ -80,9 +92,11 @@ protected void activate(BundleContext bundleContext) {
 		_bundleContext = bundleContext;
 	}
 
+	private static final String[] _CSS_FILE_NAMES = {"/css/main.css"};
+
 	private static final String[] _FILE_NAMES = {
```

Comment:
I would change this one to `_JS_FILE_NAMES` for coherence with `_CSS_FILE_NAMES`